tasks/build/browserify.js

```diff
@@ -39,7 +39,9 @@ module.exports = function(grunt) {
         browseified = browseified
           .exclude('../../docs/reference/data.json')
           .exclude('../../../docs/parameterData.json')
-          .ignore('../../translations/index.js');
+          .exclude('../../translations')
+          .ignore('i18next')
+          .ignore('i18next-browser-languagedetector');
```

Comment:
:-D

Reply:
I think there's some bug in the browserify `ignore()` method. It doesn't work for relative paths ( I tried using it and contents of the file that was supposed to be ignored were still there in the build ). `exclude()` works. 

Also, the path that is used here, must be exactly the same as that used in the import statements in the source. Which is also why it was `../../docs/` before but now it is `../../../docs` as the file importing from docs (`src/core/error_helpers.js`) became (`src/core/friendly_errors/validateParams.js`)

It took some trial and error to figure this out, but this combination seems to work :see_no_evil: